hugegraph-struct/pom.xml

```
@@ -109,9 +109,9 @@
             <version>8.1.0</version>
         </dependency>
         <dependency>
-            <groupId>org.lz4</groupId>
+            <groupId>at.yawk.lz4</groupId>
             <artifactId>lz4-java</artifactId>
```

‼️ **Critical: Incorrect GroupId for lz4-java (hugegraph-struct)**

Same issue as in hugegraph-core - using an unofficial fork `at.yawk.lz4` instead of the official `org.lz4` groupId.

```suggestion
        <dependency>
            <groupId>org.lz4</groupId>
            <artifactId>lz4-java</artifactId>
            <version>1.9.0</version>
        </dependency>
```